resources/debugServer.sh

```diff
@@ -0,0 +1,10 @@
+echo "Enter your API key: "
+read -r API_KEY
+echo "Enter your Site ID: "
+read -r SITE_ID
+mkdir v1
+mkdir v1/sites
+mkdir v1/sites/$SITE_ID
+mkdir v1/sites/$SITE_ID/prices
```

Comment:
You can do `mkdir -p v1/sites/$SITE_ID/prices`

Reply:
ooh nifty tysm!